bin/linja.in

```diff
@@ -463,7 +463,7 @@ def make_build_ninja(args):
 
         print("""rule LEAN""", file=f)
         print("""  depfile = ${DLEAN_FILE}""", file=f)
-        print("""  command = "%s" %s $in -o "${OLEAN_FILE}" """ \
+        print("""  command = "%s" %s -o ${OLEAN_FILE} $in""" \
```

Comment:
Ah, I see, `c:\users\jane mathematician\lean\...` could likely happen as well.  And come to think of it, with your french quotes notation you can also have spaces in module names.

How do you actually link your files?  I don't see any fancy `import «6.1.3.2 Divering functions»` anywhere.